litmus-portal/frontend/src/components/Sections/Settings/UserManagementTab/EditUser/index.tsx

```diff
@@ -122,41 +100,14 @@ const EditUser: React.FC<EditUserProps> = ({
                 <div>
                   <form>
                     <div className={classes.details1}>
-                      <FormControl>
-                        <Input
-                          required
-                          data-cy="changePassword"
-                          className={`${classes.pass} ${
-                            createPAssword.err ? classes.error : classes.success
-                          }`}
-                          id="outlined-adornment-password"
-                          type={
-                            createPAssword.showPassword ? 'text' : 'password'
-                          }
-                          onChange={handleCreatePassword('password')}
-                          disableUnderline
-                          endAdornment={
-                            <InputAdornment position="end">
-                              <IconButton
-                                data-cy="conVisibilty"
-                                aria-label="toggle password visibility"
-                                onClick={handleClickShowPassword}
-                                onMouseDown={handleMouseDownPassword}
-                                edge="end"
-                              >
-                                {createPAssword.showPassword ? (
-                                  <Visibility data-cy="visIcon" />
-                                ) : (
-                                  <VisibilityOff data-cy="invisIcon" />
-                                )}
-                              </IconButton>
-                            </InputAdornment>
-                          }
-                        />
-                        <InputLabel htmlFor="outlined-adornment-password">
-                          New Password
-                        </InputLabel>
-                      </FormControl>
+                      <InputField
+                        required
+                        handleChange={handleCreatePassword('password')}
+                        type="password"
+                        validationError={createPAssword.err}
```

Comment:
The password validation can be made here like suggested above with validateLength